Library/Homebrew/test/compiler_failure_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "compilers"
+
+describe CompilerFailure do
+  matcher :fail_with do |expected|
+    match do |actual|
+      actual.fails_with?(expected)
+    end
+  end
```

Comment:
Ah, of course. I did `alias_matcher :fail_with, :fails_with` instead of `alias_matcher :fail_with, :be_fails_with.